src/main/java/app/Main.java

```diff
@@ -8,6 +8,10 @@ public class Main {
 
     public static void main(String[] args) {
         EntityManagerFactory emf = HibernateConfig.getEntityManagerFactory("game_vault");
+
+        Populator populator = new Populator(emf);
+        populator.populateData();
+
```

Comment:
Populating of data should probably be removed and only added back in when necessary or at the very least be commented out so the application won't try to repopulate data each time it starts up